### DISCUSSION TITLE
setup email plugin [Oct 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4285] [Discord Threads]
> 🙋‍♂️ **ioakshay**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### step 1: click your project name on top left and click Team & Settings  
![image0](https://cdn.discordapp.com/attachments/1032982618930696274/1032983220100284476/unknown.png)  
#  
> 🙋‍♂️ **wkw**  
#### step 2: click app store  
![image0](https://cdn.discordapp.com/attachments/1032982618930696274/1032983318716751943/unknown.png)  
#  
> 🙋‍♂️ **wkw**  
#### step 3: choose a email client (SES, MailerSend, SMTP etc)  
#  
> 🙋‍♂️ **ioakshay**  
#### Thanks but on host || Port || Secure what should I put any example for me  
![image0](https://cdn.discordapp.com/attachments/1032982618930696274/1032985271601463377/unknown.png)  
#  
> 🙋‍♂️ **wkw**  
#### which version are u using  
#  
> 🙋‍♂️ **wkw**  
#### suggest to upgrade  
![image0](https://cdn.discordapp.com/attachments/1032982618930696274/1032985797953060924/unknown.png)  
#  
> 🙋‍♂️ **wkw**  
#### as i rmb there was some bugs related to email in previous versions  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1032982618930696274)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
